src/components/component.ts

```diff
@@ -65,38 +122,23 @@ export class Component {
    * @param {d3.Selection} selection.
    * @returns {Component} The calling Component.
    */
-  public anchor(selection: SimpleSelection<void>) {
+  public anchor(selection: d3.Selection<HTMLElement, any, any, any>) {
     selection = coerceExternalD3(selection);
     if (this._destroyed) {
       throw new Error("Can't reuse destroy()-ed Components!");
     }
 
-    this._isTopLevelComponent = (<Node> selection.node()).nodeName.toLowerCase() === "svg";
-
-    if (this._isTopLevelComponent) {
-      // svg node gets the "plottable" CSS class
-      this._rootSVG = selection;
-      this._rootSVG.classed("plottable", true);
-      // visible overflow for firefox https://stackoverflow.com/questions/5926986/why-does-firefox-appear-to-truncate-embedded-svgs
-      this._rootSVG.style("overflow", "visible");
```

Comment:
Yeah I feel that we shouldn't overflow the container by default since it's not expected/standard behavior; users can opt-in to that behavior by setting `overflow: visible` on the root themselves. But I can see the backcompat argument as well. Thoughts?